src/System.Net.Security/tests/FunctionalTests/SslStreamAlertsTest.cs

```diff
@@ -0,0 +1,168 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+using System.IO;
+using System.Net.Test.Common;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xunit;
+
+namespace System.Net.Security.Tests
+{
+    public class SslStreamAlertsTest
+    {
+        private const uint SEC_E_ILLEGAL_MESSAGE = 0x80090326;
+
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
```

Comment:
Are these "Windows Only", or "doesn't work on Unix (yet)?"  If it's because of the `// TODO` comments in the Unix version, these should instead be `[ActiveIssue(issueNumber, PlatformId.AnyUnix)]`